PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1728,7 +1728,7 @@ def miniAODConfigurationPre(self, process, patMetModuleSequence, pfCandCollectio
         patMetModuleSequence += getattr(process, "pfMetCHS")
         patMetModuleSequence += getattr(process, "patCHSMet")
 
-        pfTrk = cms.EDFilter("CandPtrSelector", src = cms.InputTag("packedPFCandidates"), cut = cms.string("fromPV(0) > 0 && charge()!=0"))
+        pfTrk = cms.EDFilter("CandPtrSelector", src = cms.InputTag("packedPFCandidates"), cut = cms.string("pvAssociationQuality()>=4 && charge()!=0 && vertexRef().key()==0"))
```

Comment:
if this has to be the same as in miniAOD_tools.py, please define a common reference and use it or clone from it directly in the places where these need to be the same.

Here, it looks like the package dependency chain suggests this directory is OK. So, e.g. in ` PhysicsTools/PatUtils/python/tools/trackSelectors_cff.py`
```python
chargedPackedCandsForMet = cms.EDFilter("CandPtrSelector" ... 
```

Reply:
would be great to do as well for the pfCHS
https://github.com/cms-sw/cmssw/pull/34860/files#diff-79860e2d37e9cc0867e9de61faab9b79e0283ebf986856b7a05f517e6d1d083aR1713

Reply:
The duplicate definition of pfCHS is adressed in #33885.
@lathomas: Could you take care of defining the trkMET selection as suggested by Slava (and I sync to that once you're done)?